src/test/java/net/imagej/ops/AbstractOpTest.java

```diff
@@ -235,6 +238,10 @@ private int pseudoRandom() {
 		}
 		assertFalse("More elements than expected", a.hasNext());
 	}
+	
+	public static <T> RandomAccessible<T> deinterval(RandomAccessibleInterval<T> input){
+		return Views.extendBorder(input);		
```

Comment:
@ctrueden No changes, since it seems to be the only alternative to replacing the casts.